testing/cfssl/APKBUILD

```diff
@@ -1,37 +1,34 @@
 # Contributor:
 # Maintainer:
 pkgname=cfssl
-pkgver=1.2.0
+pkgver=1.3.1
 pkgrel=0
 pkgdesc="Cloudflare PKI and TLS toolkit"
 url="https://cfssl.org"
 arch="all"
-license="BSD"
+license="BSD-2-Clause"
 makedepends="go"
 source="$pkgname-$pkgver.tar.gz::https://github.com/cloudflare/$pkgname/archive/$pkgver.tar.gz"
 builddir="$srcdir/$pkgname-$pkgver"
 
 build() {
-  cd "$srcdir"
-  export GOPATH="$PWD"
-  export GO15VENDOREXPERIMENT=1
+	cd "$srcdir"
 
-  mkdir -p src/github.com/cloudflare/
-  cp -r "$builddir" src/github.com/cloudflare/cfssl
+	export GOPATH="$PWD"
+	export GO15VENDOREXPERIMENT=1
 
-  go install -tags pkcs11 github.com/cloudflare/cfssl/cmd/...
+	mkdir -p src/github.com/cloudflare
+	cp -r "$builddir" src/github.com/cloudflare/cfssl
```

Comment:
Can't we just use a symlink created using `ln -s "$builddir" src/github.com/cloudflare/cfssl` instead?